benchmarks/graph-analytics/4.0/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM --platform=linux/amd64 cloudsuite/spark:2.4.5 as build
```

Comment:
Why is the `FROM` line like this?
It should be multi-platform now, right?

Reply:
We are doing a multi-stage build, in the first stage sbt is installed and jar file is created using sbt package. This is done in two stages as sbt is not available for all architectures. After building, we copy the jar and other files that can be run on any architecture. The second stage again imports the spark base image so it will pull the relevant arch. So, the final image is multi-platform again

Reply:
Interesting! 🤔 👍